src/python/vdo-setup-dmtest/config.yaml

```diff
@@ -0,0 +1,28 @@
+# Devices to be used by dmtest-python
+DATA_DEVICE: /dev/mapper/scratch-vdo_scratch
+METADATA_DEVICE: /dev/mapper/scratch-vdo_scratch
+
+# Dmtest-python and dependency repos
+# May be defined with URL or full path. If URL, automatic cloning will be performed.
+DMTEST_PYTHON_REPO: https://github.com/dm-vdo/dmtest-python
```

Comment:
Should this point at our fork, or the parent?  I'd assume we would want to track with the current version that everyone else is using, right?

Reply:
My understanding from discussions with Matt is that our primary intention is to use this internally. As such, it should point to our fork so that our team developers are set up to add tests to dmtest-python that we can later push upstream to the parent.